src/components/Link/Link.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <component
+    :is="tag"
+    :[linkProp]="to"
+    :class="[
+      'underline',
+      {'primary py-3 px-6 bg-primary-500 text-white active:bg-primary-700 disabled:bg-white-300': primary},
```

Comment:
These styles are very similar to the button styles - do we want to extract this into its own thing to re-use in both places? (And if so, does someone know how to do that? Found a few things via Google but didn't find a solution I really liked.)